docker/cisco-meraki/Pipfile.lock

```diff
@@ -0,0 +1,847 @@
+{
+    "_meta": {
+        "hash": {
+            "sha256": "bbc139b6de64724e7aeec1e232896f3889bcc0a839809cd7bd3b3ea2308185f7"
+        },
+        "pipfile-spec": 6,
+        "requires": {
+            "python_version": "3.10"
+        },
+        "sources": [
+            {
+                "name": "pypi",
+                "url": "https://pypi.org/simple",
+                "verify_ssl": true
+            }
+        ]
+    },
+    "default": {
+        "aiohttp": {
```

Comment:
The `aiohttp` package of all versions is vulnerable to DoS.
It has a CVE that need to be mitigated.